equalsverifier-16/src/test/java/nl/jqno/equalsverifier/internal/instantiation/vintage/reflection/RecordObjectAccessorCopyingTest.java

```diff
@@ -18,8 +18,7 @@ void copyHappyPath() {
         Object original = instantiate(SimpleRecord.class);
         Object copy = copyOf(original);
 
-        assertThat(copy).isNotSameAs(original);
-        assertThat(copy).isEqualTo(original);
+        assertThat(copy).isNotSameAs(original).isEqualTo(original);
```

Comment:
You can chain assertions on the same element; I think that looks best with each on a new line, but spotless disagrees.

Reply:
I agree it would look better that way. It's hard to find a good formatter configuration that works well in all situations...

Reply:
Trailing `//` to the rescue 😄

```suggestion
        assertThat(copy) //
                .isNotSameAs(original)
                .isEqualTo(original);
```

Reply:
Haha, that's not a bad workaround, I'll keep it in mind :smile: